OctopusDSC/DSCResources/cOctopusServerSslCertificate/cOctopusServerSslCertificate.psm1

```diff
@@ -0,0 +1,156 @@
+                CertStore       = ($certificateBinding[4] -split "`n")[0]
+            }
+        }
+    }
+
+    return ($certificateBindingsList | Where-Object {($_.AppID.Trim() -eq $ApplicationId) -and ($_.IPPort.Trim() -eq "{0}:{1}" -f "0.0.0.0", $Port) })
+}
+
+function Get-TargetResource
+{
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSDSCUseVerboseMessageInDSCResource", "")]
+    param (
+        [Parameter(Mandatory)]
+        [string] $InstanceName,
+        [ValidateSet("Present", "Absent")]
+        [string] $Ensure = "Present",
+        [ValidateNotNullOrEmpty()]
+        [Parameter(Mandatory)]
+        [string] $Thumbprint,
+        [ValidateNotNullOrEmpty()]
+        [Parameter(Mandatory)]
+        [string] $StoreName,
+        [ValidateNotNullOrEmpty()]
+        [string] $Port = "443"
+    )
+
+    $existingSSLConfig = (Get-CurrentSSLBinding -ApplicationId "{E2096A4C-2391-4BE1-9F17-E353F930E7F1}" -Port $Port -IPAddress $IPAddress)
+
+    if ($null -ne $existingSSLConfig)
+    {
+        $existingSSLPort = [int](($existingSSLConfig.IPPort).Split(":")[1])
+        $existingSSLThumbprint = $existingSSLConfig.CertificateHash.Trim()
+        $existingSSLCertificateStoreName = $existingSSLConfig.CertStore.Trim()
+        $existingEnsure = "Present"
+    }
+    else
+    {
+        $existingEnsure = "Absent"
+    }
+
+    $result = @{
+        InstanceName    = $InstanceName
+        Ensure          = $existingEnsure
+        Port            = $existingSSLPort
+        StoreName       = $existingSSLCertificateStoreName
+        Thumbprint      = $existingSSLThumbprint
+    }
+
+    return $result
+}
+
+function Test-TargetResource
+{
+    param(
+        [Parameter(Mandatory)]
+        [string] $InstanceName,
+        [ValidateSet("Present", "Absent")]
+        [string] $Ensure = "Present",
+        [ValidateNotNullOrEmpty()]
+        [Parameter(Mandatory)]
+        [string] $Thumbprint,
+        [ValidateNotNullOrEmpty()]
+        [Parameter(Mandatory)]
+        [string] $StoreName,
+        [ValidateNotNullOrEmpty()]
+        [string] $Port = "443"
+    )
+
+    $currentResource = (Get-TargetResource @PSBoundParameters)
+
+    $params = Get-ODSCParameter $MyInvocation.MyCommand.Parameters
+
+    $currentConfigurationMatchesRequestedConfiguration = $true
+    foreach ($key in $currentResource.Keys) {
+        $currentValue = $currentResource.Item($key)
+        $requestedValue = $params.Item($key)
+
+        if ($currentValue -ne $requestedValue) {
+            Write-Verbose "(FOUND MISMATCH) Configuration parameter '$key' with value '$currentValue' mismatched the specified value '$requestedValue'"
+            $currentConfigurationMatchesRequestedConfiguration = $false
+        }
+        else {
+            Write-Verbose "Configuration parameter '$key' matches the requested value '$requestedValue'"
+        }
+    }
+
+    return $currentConfigurationMatchesRequestedConfiguration
+}
+
+function Set-TargetResource
+{
```

Comment:
Can you pull opening braces onto the line above to throughout the file for consistency?